### ISSUE TITLE
TypeScript error Cannot find name 'HTMLElement' in index.d.ts

### ISSUE THREAD

**Issue by Antony92** [fastify/fastify-swagger-ui#57]
### Prerequisites

- [X] I have written a descriptive issue title
- [x] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.15.0

### Plugin version

1.8.0

### Node.js version

v18.15.0

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

13.3

### Description

Hello, running swagger-ui with typescript gives the following error when you try to build the project using `tsc -p .`

```
node_modules/@fastify/swagger-ui/types/index.d.ts:146:15 - error TS2304: Cannot find name 'HTMLElement'.
146     domNode?: HTMLElement | null | undefined;
```
The only way to fix it is by adding `dom` to the lib property in tsconfig.json

```
"compilerOptions": {
      "module": "NodeNext",
      "target": "ESNext",
      "lib": ["ESNext", "DOM"],
```

I think this is an overkill just for one type. Is this type really needed?

### Steps to Reproduce

server.ts

```
import fastify from 'fastify'
import fastifySwagger from '@fastify/swagger'
import fastifySwaggerUi from '@fastify/swagger-ui'

const server = fastify()

await server.register(fastifySwagger)
await server.register(fastifySwaggerUi, { routePrefix: '/documentation' })
```

tsconfig.json

```
{
  "compilerOptions": {
      "module": "NodeNext",
      "target": "ESNext",
      "lib": ["ESNext"],
      "moduleResolution": "NodeNext",
      "sourceMap": false,
      "rootDir": "./src",
      "outDir": "dist",
      "resolveJsonModule": true,
      "strictNullChecks": true
  },
  "exclude": [
    "src/**/*.test.ts",
  ],
  "include": [
    "src/**/*", 
  ]
}
```

package.json

```
{
  "name": "fastify-server",
  "version": "1.0.0",
  "description": "",
  "main": "dist/app.js",
  "type": "module",
  "scripts": {
    "build": "tsc -p ."
  },
  "author": "",
  "license": "MIT",
  "dependencies": {
    "@fastify/swagger": "~8.3.1",
    "@fastify/swagger-ui": "~1.8.0",
    "fastify": "~4.15.0"
  },
  "devDependencies": {
    "@types/node": "~18.15.11",
    "typescript": "~5.0.4",
  }
}

```

run `tsc -p .` or `npm run build`

**Comment by AlCabohne** [comment]
Same, so I went back to v1.3. I think this problem starts with v1.4

**Comment by mcollina** [comment]
A PR to fix this would be great.

**Comment by Antony92** [comment]
PR opened

**Comment by mcollina** [comment]
Fixed in #59
